### PR TITLE
Fix another Sass division

### LINKED ISSUE #34051: Sass deprecation warning: Using `/` for division is deprecated
dart-sass has deprecated `/` in favor of `math.div`. Compiling https://github.com/twbs/bootstrap/blob/main/scss/_variables.scss now results with many sass warnings.

```
368 │ $navbar-padding-horizontal:        floor(($grid-gutter-width / 2)) !default;
    │                                           ^^^^^^^^^^^^^^^^^^^^^^
    ╵
    node_modules/bootstrap-sass/assets/stylesheets/bootstrap/_variables.scss 368:43  @import

DEPRECATION WARNING: Using / for division is deprecated and will be removed in Dart Sass 2.0.0.

Recommendation: math.div($grid-gutter-width, 2)

More info and automated migrator: https://sass-lang.com/d/slash-div
```

### PR DESCRIPTION
**Pull request by mdo**
Closes #34051

/cc @XhmikosR

### CODE REVIEW

File: scss/mixins/_grid.scss
Comment (alpadev) [inline scss/mixins/_grid.scss:32]: I guess this has been done on purpose 🤔

**Review by XhmikosR — APPROVED** [review]
I guess it's as good as it gets for now!

File: scss/mixins/_grid.scss
Comment (XhmikosR) [reply scss/mixins/_grid.scss:32]: oops sorry missed your comment before I hit the merge button

Comment (mdo) [reply scss/mixins/_grid.scss:32]: Yeah, I had to revert it as were started getting random percentages on our widths (e.g., instead of 75%, 75.000000019% or so). Going with this for now with the shortened precision.

**Pull request merged**
